src/nupic/bindings/math.i

```diff
@@ -171,7 +149,7 @@ import_array();
 		  x_min, x_max, 1, 65535);
     return y.forPython();
   }
-  */ 
+  */
```

Comment:
Was this file missing the close for a multiline comment? I don't see an opening added.